test/hooks.ts

```diff
@@ -258,7 +258,7 @@ test('beforeRedirect is called with options and response', withServer, async (t,
 					t.is(options.url.hostname, 'localhost');
 
 					t.is(response.statusCode, 302);
-					t.is(new URL(response.url).pathname, '/redirect');
+					t.is(new URL(response.url!).pathname, '/redirect');
```

Comment:
`url` is not documented to possibly be `undefined`: https://github.com/sindresorhus/got#url-1

Reply:
Fixed.